pkgs/k1LoW/octocov/registry.yaml

```diff
@@ -13,9 +13,20 @@ packages:
       - darwin
     checksum:
       type: github_release
-      asset: checksums.txt
+      asset: checksums-{{.OS}}.txt
```

Comment:
@suzuki-shunsuke  Thank you !!!

Reply:
You're welcome.
Thank you for your great project!